core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -182,6 +187,81 @@ private SqlLibraryOperators() {
   @LibraryOperator(libraries = {MYSQL, ORACLE})
   public static final SqlFunction REGEXP_REPLACE = new SqlRegexpReplaceFunction();
 
+  @LibraryOperator(libraries = BIGQUERY)
+  public static final SqlFunction REGEXP_SUBSTR = new SqlRegexpSubstrFunction();
+
+  /**
+   * The REGEXP_EXTRACT(source_string, regex_pattern) returns the first substring in source_string
```

Comment:
Move the comments of REGEXP_EXTRACT and REGEXP_EXTRACT_ALL above their function definition